### DISCUSSION TITLE
FrozenSet and Collection Expressions.

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#7892] [General]
I noticed that FrozenSet does not currently participate in collection expressions.

Can someone please add the relevant details to this API proposal to request that it can?
https://github.com/dotnet/runtime/issues/97760

**Comment by jnm2** [comment]
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/collection-expressions#collection-builder explains the API for a type to enable support for collection expressions.
